docs/getting-started.md

```diff
@@ -18,27 +12,27 @@ These are the platform requirements to use Authorino:
 
 - [**Kubernetes**](https://kubernetes.io) server (recommended v1.20 or later), with permission to create Kubernetes Custom Resource Definitions (CRDs) (for bootstrapping Authorino and Authorino Operator)
 
-  <details markdown="1">
-    <summary>Alternative: K8s distros and platforms</summary>
+<details markdown="1">
+<summary>Alternative: K8s distros and platforms</summary>
 
-    Alternatively to upstream Kubernetes, you should be able to use any other Kubernetes distribution or Kubernetes Management Platform (KMP) with support for Kubernetes Custom Resources Definitions (CRD) and custom controllers, such as <a href="https://www.openshift.com">Red Hat OpenShift</a>, <a href="https://www.ibm.com/cloud/kubernetes-service">IBM Cloud Kubernetes Service (IKS)</a>, <a href="http://cloud.google.com/kubernetes-engine">Google Kubernetes Engine (GKE)</a>, <a href="https://aws.amazon.com/eks">Amazon Elastic Kubernetes Service (EKS)</a> and <a href="https://azure.microsoft.com/en-us/services/kubernetes-service">Azure Kubernetes Service (AKS)</a>.
-  </details>
+Alternatively to upstream Kubernetes, you should be able to use any other Kubernetes distribution or Kubernetes Management Platform (KMP) with support for Kubernetes Custom Resources Definitions (CRD) and custom controllers, such as <a href="https://www.openshift.com">Red Hat OpenShift</a>, <a href="https://www.ibm.com/cloud/kubernetes-service">IBM Cloud Kubernetes Service (IKS)</a>, <a href="http://cloud.google.com/kubernetes-engine">Google Kubernetes Engine (GKE)</a>, <a href="https://aws.amazon.com/eks">Amazon Elastic Kubernetes Service (EKS)</a> and <a href="https://azure.microsoft.com/en-us/services/kubernetes-service">Azure Kubernetes Service (AKS)</a>.
+</details>
 
 - [**Envoy**](https://www.envoyproxy.io) proxy (recommended v1.19 or later), to wire up Upstream services (i.e. the services to be protected with Authorino) and external authorization filter (Authorino) for integrations based on the reverse-proxy architecture - [example](https://github.com/kuadrant/authorino-examples#envoy)
 
-  <details markdown="1">
-    <summary>Alternative: Non-reverse-proxy integration</summary>
+<details markdown="1">
+<summary>Alternative: Non-reverse-proxy integration</summary>
 
-    Technically, any client that implements Envoy's <a href="https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/ext_authz">external authorization</a> gRPC protocol should be compatible with Authorino. For integrations based on the reverse-proxy architecture nevertheless, we strongly recommended that you leverage Envoy alongside Authorino.
-  </details>
+Technically, any client that implements Envoy's <a href="https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/ext_authz">external authorization</a> gRPC protocol should be compatible with Authorino. For integrations based on the reverse-proxy architecture nevertheless, we strongly recommended that you leverage Envoy alongside Authorino.
+</details>
```

Comment:
Does the indentation break anything? Because it was actually nesting the detail boxes inside the list items before.

<img width="1062" alt="Screenshot 2023-11-22 at 17 59 22" src="https://github.com/Kuadrant/authorino/assets/1842261/e8bfe9f4-dee3-4479-a092-b25ec9974cc5">

Now it looks a bit weird:

<img width="1062" alt="Screenshot 2023-11-22 at 17 59 33" src="https://github.com/Kuadrant/authorino/assets/1842261/4690b29a-9dcf-4be6-81ff-a29871d0d3c5">

Reply:
So it does seem like the indent matters for this. The mixing of html and markdown seems to have so many edge cases. What you will find now is the syntax highlighting does not work for the details section in editors. Or add least it does not work in my editors. It does seem to be working in the github diff.